lib/tasks/update_manual_address_entry_selected_prexisting_logs.rake

```diff
@@ -1,14 +1,20 @@
 namespace :bulk_update do
   desc "Update logs with specific criteria and set manual_address_entry_selected to true"
   task update_manual_address_entry_selected: :environment do
+    updated_lettings_logs_count = 0
+    updated_sales_logs_count = 0
     lettings_logs = LettingsLog.filter_by_year(2024)
                                .where(status: %w[in_progress completed])
                                .where(needstype: 1, manual_address_entry_selected: false, uprn: nil)
 
     lettings_logs.find_each do |log|
+      next unless log.address_line1 || log.address_line2 || log.county || log.town_or_city || log.postcode_full
```

Comment:
Can we add this into the query? I think doing it this way might take longer.

Something like `lettings_logs = LettingsLog.filter_by_year(2024).where(status: %w[in_progress completed]).where(needstype: 1, manual_address_entry_selected: false, uprn: nil).where("(address_line1 IS NOT NULL AND address_line1 != "") OR (address_line2 IS NOT NULL AND address_line2 != "") OR ...")`